privilege-management.md

```diff
@@ -181,34 +184,62 @@ SHOW GRANTS FOR `rw_user`@`192.168.%`;
 +------------------------------------------------------------------+
 ```
 
+### Dynamic privileges
+
+<!-- include a "since" for the version number after TiDB 5.0 --> TiDB features support dynamic privileges, a feature borrowed from MySQL 8.0. Dynamic privileges are intended to replace the `SUPER` privilege by implementing more fine-grained access to certain operations. For example, with dynamic privileges it is possible to grant access to perform `BACKUP` and `RESTORE` operations but not `SHOW PROCESSLIST`.
+
+Dynamic privileges include:
+
+* `BACKUP_ADMIN`
+* `ROLE_ADMIN`
+* `CONNECTION_ADMIN`
+* `SYSTEM_VARIABLES_ADMIN`
+
+The full set of dynamic privileges can be observed with `SHOW PRIVILEGES`. Because plugins are permitted to add new privileges, it is possible that the list of privileges that are assignable will differ based on your TiDB installation.
```

Comment:
```suggestion
To see the full set of dynamic privileges, execute the `SHOW PRIVILEGES` statement. Because plugins are permitted to add new privileges, the list of privileges that are assignable might differ based on your TiDB installation.
```